Sources/SymbolKit/SymbolGraph/LineList/SourceRange/SourceRange+Position.swift

```diff
@@ -24,5 +24,13 @@ extension SymbolGraph.LineList.SourceRange {
          The zero-based *byte offset* into a line.
          */
         public var character: Int
+        
+        /**
+         Create a new cursor position with the given line number and character offset.
```

Comment:
nit, same for other comments, to align with the general commenting style of Swift-DocC:
```suggestion
         Creates a new cursor position with the given line number and character offset.
```

Reply:
Good catch- thanks! 

Addressed here: fd42b5d.